HW02_B_DanielKrochmal.Rmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "HW02_B_Graph-Mimic"
+author: "Daniel Krochmal"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+
+#Create plot base and add main title
+ggplot(data = diamonds) +
+  geom_bar(mapping = aes (x = cut, fill = clarity), position = "dodge") +
+  ggtitle("My Diamond Collection") +
+  
+  #Add title and subtitle
+  labs(
+    title = "My Diamond Collection",
+    subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarify of diamond"
+  ) +
+  
+  #Rename x and y labels
+  xlab("Diamond Cut") +
```

Comment:
As you pointed out on my HW02_B, x and y axis labels can be named within a single labs function. However, the xlab and ylab functions you used here seem nice and simple. They would be great for scenarios where everything does not need a complete relabeling.